crates/turbopack-ecmascript/src/chunk/optimize.rs

```diff
@@ -289,7 +289,7 @@ async fn merge_by_size(chunks: &[EcmascriptChunkVc]) -> Result<Vec<EcmascriptChu
                     merged.push(current.pop().unwrap());
                 } else {
                     merged.push(merge_chunks(*current.first().unwrap(), &current).await?);
-                    current.clear();
+                    current.truncate(0);
```

Comment:
Both methods' documentation read:
> Note that this method has no effect on the allocated capacity of the vector.

Reply:
hmm... you are right... seems like I misread that